tests/src/Unit/StreamContextFactoryTest.php

```diff
@@ -47,4 +47,56 @@ public function testProxyStreamContext()
         $proxyAuthHeader = 'Proxy-Authorization: Basic ' . base64_encode($proxy['login'] . ':' . $proxy['password']);
         $this->assertContains($proxyAuthHeader, $options['http']['header']);
     }
+
+    public function testAuthorizationHeaderShouldBeDefined()
+    {
+        $config = new Config(array(
+            'inputFile' => null,
+            'outputDir' => null,
+            'soapClientOptions' => array(
+                'authentication' => SOAP_AUTHENTICATION_BASIC,
+                'login' => 'user',
+                'password' => 'secret'
+            )
+        ));
+
+        $factory = new StreamContextFactory();
+        $resource = $factory->create($config);
+
+        $options = stream_context_get_options($resource);
+
+        $this->assertArrayHasKey('http', $options);
+        $this->assertArrayHasKey('header', $options['http']);
+        $this->assertContains('Authorization: Basic dXNlcjpzZWNyZXQ=', $options['http']['header']);
```

Comment:
This as well the subsequent test contains the actual base64 encoded value instead of inserted a runtime computed value as in the previous tests. Do you think this is a better way to handle it?